front/pages/edit.vue

```diff
@@ -20,7 +20,7 @@
             h-auto
           "
         >
-          <div>
+          <div class="mt-2">
```

Comment:
Let's just remove this and replace `md:mx-4 md:mb-4` (L19) with `md:m-4`, that's way cleaner